pokemongo_bot/event_handlers/telegram_handler.py

```diff
@@ -1,29 +1,25 @@
 # -*- coding: utf-8 -*-
 from pokemongo_bot.event_manager import EventHandler
 from pokemongo_bot.base_dir import _base_dir
-import json
-import os
 import time
 import telegram
 import thread
 import re
-from pokemongo_bot.datastore import Datastore
 import pprint
+from pokemongo_bot.datastore import Datastore
 from pokemongo_bot import inventory
-
+from telegram.utils import request
 
 DEBUG_ON = False
 
-class FileIOException(Exception):
-    pass
-
 class TelegramClass:
 
     update_id = None
-
+    session = None
 
     def __init__(self, bot, master, pokemons, config):
         self.bot = bot
+        request.CON_POOL_SIZE = 16
         with self.bot.database as conn:
             # initialize the DB table if it does not exist yet
             initiator = TelegramDBInit(bot.database)
```

Comment:
@sohje not part of this PR, but it's in both classes due to the order in which they *might* be called. Both classes try to access db tables and will generate an error if tables don't exist.